src/overlays/actors/ovl_Item_Etcetera/z_item_etcetera.h

```diff
@@ -6,11 +6,30 @@
 
 struct ItemEtcetera;
 
+typedef void (*ItemEtceteraActionFunc)(struct ItemEtcetera*, GlobalContext*);
+typedef void (*ItemEtceteraDrawFunc)(struct ItemEtcetera*, GlobalContext*);
+
 typedef struct ItemEtcetera {
     /* 0x0000 */ Actor actor;
-    /* 0x014C */ char unk_14C[0x14];
+    /* 0x014C */ ItemEtceteraActionFunc actionFunc_14C;
+    /* 0x0150 */ s16 drawId;
+    /* 0x0152 */ s16 getItemId;
+    /* 0x0154 */ u8 objBankIndex;
+    /* 0x0158 */ ActorFunc currentDrawRoutine; // gets set to this->actor.draw
```

Comment:
yeah im thinking just drawFunc for the name, and the comment is implied at that point and can be removed

Reply:
Okay I'll change to drawFunc